augur/refine.py

```diff
@@ -241,11 +241,27 @@ def run(args):
     if args.divergence_units=='mutations-per-site': #default
         pass
     elif args.divergence_units=='mutations':
-        L = tt.seq_len
-        for node in node_data['nodes']:
+        if not args.timetree:
+            tt.infer_ancestral_sequences()
+        nuc_map = profile_maps['nuc']
```

Comment:
If we use the "[nuc_nogap](https://github.com/neherlab/treetime/blob/3b2aab3711127338a288395226952cf52be0fae1/treetime/seq_utils.py#L45-L63)" map here instead of the "nuc" map, won't that allow us to remove the checks for "-" and "N" characters in the `different` function below? It seems like this map will ignore those gap bases in the calculations below.

Reply:
my main reason for doing this check for `N` and `-` was speed (without profiling). I think we should use the same alphabet as the build though.

Reply:
Ah, ok. Thank you for clarifying this!